pagopa/io-app
# chore: [IOPLT-326] Change reference on io-react-native-zendesk to load from zendesk

## Short description
This PR changes the installation reference of the io-react-native-zendesk library from github to npm release

## How to test
Nothing should be changed, a part from the installation point of the library.
Open a Zendesk help request to check anything's fine.
